.github/workflows/fly-deploy-prod.yml

```
@@ -0,0 +1,61 @@
+    inputs:
+      version:
+        description: "Version to deploy (without v prefix)"
+        required: true
+        type: string
+        default: 'manual'
+
+jobs:
+  deploy-honcho-prod-image:
+    name: Deploy Honcho Image (Production Environment)
+    runs-on: ubuntu-latest
+    concurrency: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)